src/services/auth.js

```diff
@@ -2,8 +2,9 @@ import * as firebase from 'firebase/app';
 import 'firebase/auth';
 
 import SIGNUP_ENDPOINT from '@Constants/endpoints';
+import { config } from '@Config';
 
-const SIGN_UP_URL = `${process.env.REACT_APP_API_BASE_URL}${SIGNUP_ENDPOINT}`;
+const SIGN_UP_URL = `${config.apiBaseURL}${SIGNUP_ENDPOINT}`;

```

Comment:
is this even used? we should change this to the autogen stuff (can be for later pr)

Reply:
Yeah since we have autogen this is not needed anymore, including the function that actually uses this url. Will remove in some other pr